docs/docsite/rst/reference_appendices/faq.rst

```diff
@@ -754,6 +754,45 @@ In such environments we recommend securing around Ansible's execution but still
 This is something that Tower/AWX excels at by allowing administrators to set up RBAC access to inventory, along with managing credentials and job execution.
 
 
+.. _complex_configuration_validation:
+
+The 'validate' option is not enough for my needs, what do i do?
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Many Ansible modules that create or update files have a ``validate`` option that allows you to abort the update if the validation command fails,
+this uses the temporary file Ansible creates before doing the final update. In many cases this does not work since the validation tools
```

Comment:
```suggestion
This uses the temporary file Ansible creates before doing the final update. In many cases this does not work since the validation tools
```